submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/ProjectsApiTest.kt

```diff
@@ -75,7 +79,22 @@ internal class ProjectsApiTest(private val tempFolder: TemporaryFolder) : BaseIn
 
         @Test
         fun `submit project`() {
-            // TODO create a project and verify it exitst with the proper access tags
+            val project = tsv {
+                line("Submission", "AProject")
+                line("Title", "A Project")
+                line()
+
+                line("Project")
+            }
+
+            val response = webClient.submitProject(tempFolder.createFile("a-project.tsv", project.toString()))
+            assertThat(response).isNotNull
+            assertThat(response.statusCode).isEqualTo(HttpStatus.OK)
+
+            val submittedProject = submissionRepository.getExtendedByAccNo("AProject")
+            assertThat(submittedProject.accNo).isEqualTo("AProject")
+            assertThat(submittedProject.title).isEqualTo("A Project")
+            assertThat(tagsDataRepository.existsByName("AProject")).isTrue()
```

Comment:
don't we need to validate that access tag is created for project

Reply:
Done